<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>750</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Invoice Details</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::InputMethodHint::ImhNone</set>
  </property>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>71</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>LOGO</string>
   </property>
  </widget>
  <widget class="QLabel" name="companyName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tesla Inc</string>
   </property>
  </widget>
  <widget class="QLabel" name="companyPhone">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(877) 798-3752</string>
   </property>
  </widget>
  <widget class="QLabel" name="companyEmail">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>apacpress@tesla.com</string>
   </property>
  </widget>
  <widget class="QLabel" name="companyAddress">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1 Tesla Road, Austin, TX 78725</string>
   </property>
  </widget>
  <widget class="QLabel" name="productNameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>14</pointsize>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Product Name:</string>
   </property>
   <property name="buddy">
    <cstring>productCategory</cstring>
   </property>
  </widget>
  <widget class="QComboBox" name="productCategory">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Product 101</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Product 102</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Product 103</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Product 104</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Product 105</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Product 106</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Product 107</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Product 108</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="priceLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>190</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>14</pointsize>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Price:</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="salesInvoiceLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sales Invoice</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>150</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="quantityLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>230</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>14</pointsize>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Quantity:</string>
   </property>
   <property name="buddy">
    <cstring>quantityProduct</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="quantityProduct">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>230</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::InputMethodHint::ImhDigitsOnly</set>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>631</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="addProductButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>320</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add Product</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>621</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>125</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Product</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Price</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Quantity</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Discount</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Amount</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="generateInvoiceButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>690</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Generate Invoice</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="clientNameInfo">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::InputMethodHint::ImhDigitsOnly</set>
   </property>
  </widget>
  <widget class="QLabel" name="clientName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>14</pointsize>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Client Name:</string>
   </property>
   <property name="buddy">
    <cstring>clientNameInfo</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="amountLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>14</pointsize>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Amount:</string>
   </property>
  </widget>
  <widget class="QLabel" name="amountValue">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>14</pointsize>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="discountValue">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>270</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::InputMethodHint::ImhDigitsOnly</set>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="discountLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>270</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>14</pointsize>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Discount:</string>
   </property>
   <property name="buddy">
    <cstring>discountValue</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="priceValue">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>190</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>14</pointsize>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>productCategory</tabstop>
  <tabstop>quantityProduct</tabstop>
  <tabstop>discountValue</tabstop>
  <tabstop>clientNameInfo</tabstop>
  <tabstop>addProductButton</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>generateInvoiceButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
